go/logic/applier.go

```diff
@@ -179,7 +180,7 @@ func (this *Applier) CreateGhostTable() error {
 		sql.EscapeName(this.migrationContext.DatabaseName),
 		sql.EscapeName(this.migrationContext.GetGhostTableName()),
 	)
-	if _, err := sqlutils.ExecNoPrepare(this.db, query); err != nil {
+	if _, err := sqlutils.ExecNoPrepare(this.db, this.migrationContext.DisableBinLogForQuery(query)); err != nil {
```

Comment:
When the user opts out, I wrap almost all queries in a `set sql_log_bin=off; statement; set sql_log_bin=on`